README.md

```diff
@@ -39,6 +41,47 @@ MAP=/destination/project.qgs&
 REQUEST=GetCapabilities
 ```
 
+* Example of using SOURCES with tile URI:
+
+```
+http://localhost:81/qgis?
+SERVICE=MAPCOMPOSITION&
+PROJECT=/destination/project.qgs&
+SOURCES=type%253Dxyz%2526url%253Dhttp%25253A%2F%2Fa.tile.osm.org%2F%25257Bz%25257D%2F%25257Bx%25257D%2F%25257By%25257D.png;/path/1.shp;/path/2.geojson;/path/3.asc&
+NAMES=Basemap;My layer 1;MyLayer 2;Layer 3&
+OVERWRITE=true
+```
+
+In the sample request above, note that the datasource: ```type%253Dxyz%2526url%253Dhttp%25253A%2F%2Fa.tile.osm.org%2F%25257Bz%25257D%2F%25257Bx%25257D%2F%25257By%25257D.png``` were urlquoted twice.
```

Comment:
You could use single quote for inline code ;-)

Reply:
Hahaha... my habit, I guess.